manifester/commands.py

```diff
@@ -1,18 +1,23 @@
 import click
 
-from manifester import logger as mlog
-from manifester.settings import settings
 from manifester import Manifester
 
+
 # To do: add a command for returning subscription pools
 @click.group
 def cli():
     pass
 
 
 @cli.command()
-@click.option("--manifest_category", type=str, help="Category of manifest (golden_ticket or robottelo_automation by default)")
-@click.option("--allocation_name", type=str, help="Name of upstream subscription allocation")
+@click.option(
+    "--manifest_category",
```

Comment:
Would it be possible to change underscore to dash without changing other parts of the code?
Dashes feel more standard to me than underscores in arg names.

Reply:
Gonna call this not in scope, but a great suggestion to apply to all the CLI args defined in this module.